### PR TITLE
Add doc for Windows Debugging

### PR DESCRIPTION
**Pull request by okkenator**

### CODE REVIEW

File: docs/WindowsNative.md [hunk 1]
```diff
@@ -55,6 +58,31 @@ and you should be done!
...
+            "localRoot": "${workspaceRoot}",
+            "remoteRoot": "C:\\Users\\${username}\\compiler-explorer"
+        }
+    ]
+}
+```
+
+Launch CE with `npm run-script debugger` to have node listen on port 9229. 
+
+Because you can only attach to the process, as opposed to launching the process, in order to debug startup code you need to attach while npm is starting up.  The `debugger` script also enables `debug` logging level so debug print statements can be seen during the CE startup and run.
```
Comment (partouf) [inline docs/WindowsNative.md:85]: I don't think this is true. Can you rephrase to "Because this only attached to a running process, as opposed to ..."

**Review by partouf — CHANGES_REQUESTED** [review]
Thanks for the PR! Minor nitpick on the text. And I'll remove the package-lock changes before merging.

**Comment by partouf** [comment]
Thanks!

File: docs/WindowsNative.md [hunk 2]
```diff
@@ -36,6 +36,9 @@ then run
 
 ```bat
 npm install
+npm install webpack -g
```
Comment (apmorton) [inline docs/WindowsNative.md:39]: @okkenator can you comment on why these three npm commands were added?  I have just installed node on windows to test this and found these were not required.  I can run the following commands and be up and running without issues.

```
git clone https://github.com/compiler-explorer/compiler-explorer.git
cd compiler-explorer
npm install
npm start
```

I don't think we should be suggesting global installation of webpack in our docs unless there is a very good reason.

Comment (okkenator) [reply docs/WindowsNative.md:39]: Yes, I was not able to get it to work without that so I thought everyone else would need the same.  As I am new to using npm, it took me a little while to figure out.  I do not remember the exact error I was getting.  Maybe it should be listed as an option only if people have problems, or you could remove it as it is not ce specific but rather npm.
